assets/js/base/components/price-slider/price-input.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import { Fragment } from '@wordpress/element';
+
+const PriceInput = ( { disabled, onBlur, onChange, minPrice, maxPrice } ) => {
+	return (
+		<Fragment>
+			<input
+				type="text"
+				size="5"
+				className="wc-block-price-filter__amount wc-block-price-filter__amount--min wc-block-form-text-input"
+				aria-label={ __(
+					'Filter products by minimum price',
+					'woo-gutenberg-products-block'
+				) }
+				onChange={ onChange }
+				onBlur={ onBlur }
+				disabled={ disabled }
+				value={ minPrice }
+			/>
+			<input
+				type="text"
+				size="5"
+				className="wc-block-price-filter__amount wc-block-price-filter__amount--max wc-block-form-text-input"
+				aria-label={ __(
+					'Filter products by maximum price',
+					'woo-gutenberg-products-block'
+				) }
+				onChange={ onChange }
+				onBlur={ onBlur }
+				disabled={ disabled }
+				value={ maxPrice }
+			/>
+		</Fragment>
+	);
+};
+
+PriceInput.propTypes = {
+	/**
+	 * Is the text input disabled?
+	 */
+	disabled: PropTypes.bool,
+	/**
+	 * Callback fired on input.
+	 */
+	onBlur: PropTypes.func.isRequired,
+	/**
+	 * Callback fired on input.
+	 */
+	onChange: PropTypes.func.isRequired,
+	/**
+	 * Min price to display.
+	 */
+	minPrice: PropTypes.string.isRequired,
+	/**
+	 * Max price to display.
+	 */
+	maxPrice: PropTypes.string.isRequired,
+};
```

Comment:
Are these strings because they will contain a currency symbol?  I wonder if it'd be good do document that here.  I realize this is currently an internal only component so we don't want to worry too much about internals but for our future selves or whoever maintains would be good to be clear why it's a string?